packages/opentelemetry-tracer/src/instrumentation/PluginLoader.ts

```diff
@@ -98,7 +101,8 @@ export class PluginLoader {
 
         // Expecting a plugin from module;
         try {
-          const plugin: Plugin = require(moduleName).plugin;
+          // tslint:disable-next-line:no-any
+          const plugin: Plugin<any> = require(moduleName).plugin;
```

Comment:
What happens if you give it `unknown` here rather than `any`?

Reply:
Compilation Error: `Type 'unknown' is not assignable to type 'T'`